src/app/multi-editor/multi-editor.component.ts

```diff
@@ -238,6 +252,10 @@ export class MultiEditorComponent implements OnInit {
 
   filterRecords(newFilterExpressionArray) {
```

Comment:
add types for this one, an I think the way you use it you don't need a parameter since you're passing a class property.

Reply:
Im also passing the event in the template